docs/sphinx/source/user_guide/modeling_topics/temperature.rst

```diff
@@ -0,0 +1,93 @@
++===========================================+========+============+================+=====================+============+================+
+| :py:func:`~pvlib.temperature.faiman`      | either |            | ✓              | ✓                   | ✓          |                |
++-------------------------------------------+--------+------------+----------------+---------------------+------------+----------------+
+| :py:func:`~pvlib.temperature.faiman_rad`  | either |            | ✓              | ✓                   | ✓          | ✓              |
++-------------------------------------------+--------+------------+----------------+---------------------+------------+----------------+
+| :py:func:`~pvlib.temperature.fuentes`     | cell   | ✓          | ✓              | ✓                   | ✓          |                |
++-------------------------------------------+--------+------------+----------------+---------------------+------------+----------------+
+| :py:func:`~pvlib.temperature.noct_sam`    | cell   |            | ✓              | ✓                   | ✓          |                |
++-------------------------------------------+--------+------------+----------------+---------------------+------------+----------------+
+| :py:func:`~pvlib.temperature.pvsyst_cell` | cell   |            | ✓              | ✓                   | ✓          |                |
++-------------------------------------------+--------+------------+----------------+---------------------+------------+----------------+
+| :py:func:`~pvlib.temperature.ross`        | cell   |            | ✓              | ✓                   |            |                |
++-------------------------------------------+--------+------------+----------------+---------------------+------------+----------------+
+| :py:func:`~pvlib.temperature.sapm_cell`   | cell   |            | ✓              | ✓                   | ✓          |                |
++-------------------------------------------+--------+------------+----------------+---------------------+------------+----------------+
+| :py:func:`~pvlib.temperature.sapm_module` | module |            | ✓              | ✓                   | ✓          |                |
++-------------------------------------------+--------+------------+----------------+---------------------+------------+----------------+
+
+
+Model parameters
+----------------
+
+Some temperature model functions provide default values for their parameters,
+and several additional sets of temperature model parameter values are
+available in :py:data:`pvlib.temperature.TEMPERATURE_MODEL_PARAMETERS`.
+However, these generic values may not be suitable for all modules.
+
+Module-specific values can be obtained via testing, for example following
+the IEC 61853-2 standard.
+
+Currently, pvlib provides no functionality for fitting parameter values
+using measured temperature.
```

Comment:
I think we should at least mention the conversion capabilities in `GeneralLinearModel`